lib/zendesk_apps_support/assets/installed.js.erb

```diff
@@ -0,0 +1,15 @@
+(function() {
+  <% appsjs.each do | appjs | %>
```

Comment:
lol +1 it took a second to figure out what this meant. But that could also be the vacation